docs/getting-started.mdx

```diff
@@ -105,15 +105,54 @@ They are documented in the [API Reference](pathname:///api/v7/classes/Sequelize.
 
 ### Testing the connection
 
-You can use the `.authenticate()` function to test if the connection is OK:
+You can use the `.authenticate()` function to test if the connection is OK, it runs `SELECT 1+1 AS result` query to test:
 
 ```js
+const sequelize = new Sequelize("db", "user", "pass");
+
 try {
   await sequelize.authenticate();
   console.log('Connection has been established successfully.');
 } catch (error) {
   console.error('Unable to connect to the database:', error);
 }
+
+// or using Promise
+sequelize
```

Comment:
I'm opposed to this change. It adds noise to an already very complex documentation. It's not on us to teach people how to use promises. Let's exclusively stick to async/await in our docs